test/library/gui/dialogs.lua

```diff
@@ -0,0 +1,263 @@
+config.mode = 'fortress'
```

Comment:
I think this line can be removed - I removed it on my end, and the tests run without issue on the title screen.

Reply:
If I remove it, the test might cause problems in some CI pipeline (or something, I don't know)

https://github.com/DFHack/dfhack/blob/e1f0e94f0f04729d5b4a0e4e70a8c2266d9b0928/ci/test.lua#L286-L287

https://github.com/DFHack/dfhack/blob/e1f0e94f0f04729d5b4a0e4e70a8c2266d9b0928/ci/test.lua#L46-L48

Reply:
Ok, well that is probably because fortress mode tests _don't run_ in CI currently. If it is failing in CI, that's something that should be fixed. Do you know where the error was occurring? I'll see if I can track it down, but if not, I am inclined to remove this line and see what happens.

Reply:
The line in question was added at the top of 70d088c763138ee3199080387e5bbfe68feeeae9 and hasn't been changed since, so I don't know how the lack of it could have triggered a CI failure. I'll try removing it on your branch.

Reply:
> The line in question was added at the top of [70d088c](https://github.com/DFHack/dfhack/commit/70d088c763138ee3199080387e5bbfe68feeeae9) and hasn't been changed since, so I don't know how the lack of it could have triggered a CI failure. I'll try removing it on your branch.

sorry for confusion, I haven't actually experienced any problems, I just assumed something somewhere would go wrong.

But now that I think about it, you are right, a dialog window should work regardless of mode.